src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -873,7 +886,25 @@ export function GraphWrapper({
 		onChangeColumns?.(graphColumnsConfig);
 	};
 
+	// dirty trick to avoid mutations on the GraphContainer side
+	const fixedExcludeRefsById = useMemo(() => ({ ...excludeRefsById }), [excludeRefsById]);
 	const handleOnToggleRefsVisibilityClick = (_event: any, refs: GraphRefOptData[], visible: boolean) => {
+		if (!visible) {
+			document.getElementById('test')?.animate(
```

Comment:
Can we rename it from 'test' to something more readable now that this is an official PR?